cves/kernel/CVE-2019-19057.yml

```diff
@@ -358,8 +368,8 @@ forgotten_check:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: The check is there but they didn't do the right things, freeing memory, afterwards.
```

Comment:
The commas with the "freeing memory" feels a little clunky to read, I think you can simplify it down to "The check is there but they did not free the memory afterwards" or something along those lines